library/item/item_extension/src/testmod/java/org/quiltmc/qsl/item/extension/test/ArmorKnockbackTest.java

```diff
@@ -0,0 +1,83 @@
+package org.quiltmc.qsl.item.extension.test;
+
+import static org.quiltmc.qsl.item.extension.test.BowsTest.MOD_ID;
+
+import java.util.List;
+
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.client.item.TooltipContext;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.item.ArmorItem;
+import net.minecraft.item.ArmorMaterial;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.recipe.Ingredient;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.Registry;
+import net.minecraft.sound.SoundEvent;
+import net.minecraft.sound.SoundEvents;
+import net.minecraft.text.Text;
+import net.minecraft.util.Identifier;
+import net.minecraft.util.Rarity;
+import net.minecraft.world.World;
+
+import org.quiltmc.loader.api.ModContainer;
+import org.quiltmc.qsl.base.api.entrypoint.ModInitializer;
+
+public class ArmorKnockbackTest implements ModInitializer {
+	private static final ArmorMaterial KNOCKBACK_RES_ARMOR = new ArmorMaterial() {
```

Comment:
Hm, I wonder if this should be renamed to "Knockback Resolution Armor" or "Anti-Knockback Armor"

Reply:
I don't really think it's worth it - none of the items in any of the item ext. testmods even have translation entries anyway :P

Reply:
oh well, that's fair enough